cablelabs/snaps-kubernetes
# Rook issue 161

#### What does this PR do?
Fixes #161 
This provides the initial support for installing Rook onto K8s
#### Do you have any concerns with this PR?
no, except that this feature probably requires more configuration but it works for now. We also need to determine how to use Rook PVs properly
#### How can the reviewer verify this PR?
Run CI and check that the rook operator, cluster items, and Persistent Volume objects have been properly installed and created
#### Any background context you want to provide?
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
yes
- Does the documentation need an update?
done
- Does this add new Python dependencies?
no
- Have you added unit or functional tests for this PR?
yes
- Does this patch update any configuration files?
yes, adds one section on creating Rook Persistent Volumes.
